code/modules/reagents/reagents/food-Drinks.dm

```diff
@@ -110,6 +115,7 @@
 	reagent_state = LIQUID
 	nutriment_factor = 0.8
 	color = "#792300"
+	taste_tag = list(TASTE_SALTY)
```

Comment:
The only problem with this is if people start taking salt to satisfy their need for a salty drink.

Reply:
hummm i see. on a note people already take 120u of sugar around moebius for sanity
. lemme fix this salty thing tho